src/components/navbar/Navbar.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+import { Layout, Menu } from 'antd'
+
+import routes from '../../config/routes'
+import './style.css'
+
+const renderLinks = () => {
```

Comment:
Use the Class syntax like containers or complexe components. Put this method inside of it and don't forget to `.bind` it.